mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_project_big_query_export_config_test.go

```diff
@@ -0,0 +1,148 @@
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"update_time", "project"},
+			},
+			{
+				Config: testAccSecurityCenterV2ProjectBigQueryExportConfig_update(context),
+			},
+			{
+				ResourceName:            "google_scc_v2_project_scc_big_query_exports.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"update_time", "project"},
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterV2ProjectBigQueryExportConfig_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+
+resource "google_bigquery_dataset" "default" {
+  dataset_id                  = "%{dataset_id}"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "US"
+  default_table_expiration_ms = 3600000
+  default_partition_expiration_ms = null
+
+  labels = {
+    env = "default"
+  }
+
+  lifecycle {
+	ignore_changes = [default_partition_expiration_ms]
+  }
+}
+
+resource "time_sleep" "wait_1_minute" {
+	depends_on = [google_bigquery_dataset.default]
+	create_duration = "3m"
+}
+
+resource "google_scc_v2_project_scc_big_query_exports" "default" {
+  name		   = "%{name}"
+  big_query_export_id    = "%{big_query_export_id}"
+  project      = "%{project}"
+  dataset      = "%{dataset}"
+  location     = "global"
+  description  = "Cloud Security Command Center Findings Big Query Export Config"
+  filter       = "state=\"ACTIVE\" AND NOT mute=\"MUTED\""
+
+  lifecycle {
+	ignore_changes = [name]
+  }
+
+  depends_on = [time_sleep.wait_1_minute]
+}
+
+resource "time_sleep" "wait_for_cleanup" {
+	create_duration = "3m"
+	depends_on = [google_scc_v2_project_scc_big_query_exports.default]
+}
+`, context)
+}
+
+func testAccSecurityCenterV2ProjectBigQueryExportConfig_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+
+resource "google_bigquery_dataset" "default" {
+  dataset_id                  = "%{dataset_id}"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "US"
+  default_table_expiration_ms = 3600000
+  default_partition_expiration_ms = null
+
+  labels = {
+    env = "default"
+  }
+
+  lifecycle {
+	ignore_changes = [default_partition_expiration_ms]
+  }
+}
+
+resource "google_scc_v2_project_scc_big_query_exports" "default" {
+  name		   = "%{name}"
+  big_query_export_id    = "%{big_query_export_id}"
+  project      = "%{project}"
+  dataset      = "%{dataset}"
+  location     = "global"
+  description  = "SCC Findings Big Query Export Update"
+  filter       = "state=\"ACTIVE\" AND NOT mute=\"MUTED\""
+
+  lifecycle {
+	ignore_changes = [name]
+  }
+}
+
+resource "time_sleep" "wait_for_cleanup" {
+	create_duration = "3m"
+	depends_on = [google_scc_v2_project_scc_big_query_exports.default]
+}
```

Comment:
Why is time_sleep needed this doesn't seem right?

Reply:
This is to clear up the resources, I was getting the error regarding the cleanup

Reply:
I believe this should be fixed with the dependent relationship between the dataset and the other resource being encoded in config. Feel free to try.

Reply:
It's working without the cleanup now, removed it. Thanks.